support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -736,6 +744,8 @@ export function CheckoutComponent({
 				thankYouUrlSearchParams.set('product', productKey);
 				thankYouUrlSearchParams.set('ratePlan', ratePlanKey);
 				promoCode && thankYouUrlSearchParams.set('promoCode', promoCode);
+				userType && thankYouUrlSearchParams.set('userType', userType);
```

Comment:
Will this filter out `userType=undefined` at this point or later?

Reply:
This will filter it out here. But we have to assume it could be null when the search params are read on the next page